### DISCUSSION TITLE
stop syncing when using metered connections

### DISCUSSION THREAD

**Discussion by nicopace** [ente-io/ente#1117] [Enhancements]
Some phones allow the user to define a connection as metered... Meaning that it is similar to a mobile internet where every GB counts.
Would be good for ente to respect this.
Thanks!

**Comment by ua741** [comment]
You can prevent uploads on a metered connection by turning off the following option: `Backup` -> `Backup Settings` -> `Backup over mobile data`.

Is this what you were referring to? If not, could you please elaborate further?

**Reply by nicopace** [reply]
I I have that turned off.
In my Android I can configure a wifi connection as metered.
It is described in this page: https://support.google.com/android/answer/9654714?hl=en#zippy=%2Ccontrol-data-use-with-metered-wi-fi

**Reply by ua741** [reply]
Makes sense. 

---
We use `connectivity_plus` package to fetch connection status. Currently, it doesn't provide a way to detect metered connection: https://github.com/fluttercommunity/plus_plugins/issues/612.  We will fix it as soon as the linked issue is resolved.
